app/assets/javascripts/darkswarm/services/enterprise_registration_service.js.coffee

```diff
@@ -55,6 +55,10 @@ angular.module('Darkswarm').factory "EnterpriseRegistrationService", ($http, Reg
       ).catch((response) ->
         Loading.clear()
         alert(t('failed_to_update_enterprise_unknown'))
+        if response.data.errors.instagram
+          igErr = document.querySelector("#instagram-error")
```

Comment:
Not generic, but should do the job. I'm wondering if we wanted to make a generic solution, or make a quick but a bit dirty solution as we don't want to spend to much time on it.

Reply:
yes, it is a bit dirty. But not sure how to make it more clean :sweat_smile: 
If you could point me to a commit or an article that would help me, I will try to make it more clean. Just don't know angular well enough

Reply:
I was also wondering if we should support other fields here, but it seems this is the only one with special validation.